main.py

```diff
@@ -0,0 +1,100 @@
+import ee
+import argparse
+import geemap
+from geopy.geocoders import Nominatim
+from geopy.exc import GeopyError
+from decouple import config
+
+HEAT_MAP_GEE_PROJECT = config("GEE_PROJECT")
+ee.Initialize(project=HEAT_MAP_GEE_PROJECT)
+
+def get_city_coordinates()->list:
+    """Asks user for city"""
+    try: 
+        city_name = str(input("Enter the city urban center you want to view: "))
```

Comment:
This is a side effect, making the function harder to test. I see you get the city from the command line parsing below, so I would pass it in as argument to this function.